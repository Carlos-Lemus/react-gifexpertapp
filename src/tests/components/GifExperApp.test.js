import { shallow } from "enzyme";
import GifExpertApp from "../../GifExpertApp";

describe("Probando el componente GifExpertApp", () => {

    const wrapper = shallow( <GifExpertApp /> );

    test('Debe de mostrar el componente de GifExpertApp', () => {

        expect( wrapper ).toMatchSnapshot();

    })
    

});