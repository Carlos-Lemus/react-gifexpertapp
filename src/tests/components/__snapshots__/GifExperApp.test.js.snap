// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando el componente GifExpertApp Debe de mostrar el componente de GifExpertApp 1`] = `
<Fragment>
  <h2>
    Gif Expert App
  </h2>
  <hr />
  <AddCategory
    setCategories={[Function]}
  />
  <ol>
    <GifGrid
      category="One Punch Man"
      key="One Punch Man"
    />
  </ol>
</Fragment>
`;
